libsigopt/views/view.py

```diff
@@ -103,7 +113,11 @@ def get_relevant_expected_improvement(predictor):
 
 
 class View(object):
-  view_name = None
+  view_name: str
+  optimized_metrics_index: list | _UNSET_CLS
+  optimized_metrics_objectives: list | _UNSET_CLS
```

Comment:
can't we just make these to be `| None` and `assert is not None` ?

Reply:
I would prefer that, however I left it as-is